src/features/schedule/ScheduleCardCourses.tsx

```diff
@@ -83,11 +107,14 @@ const ScheduleCardCourses = (props: ScheduleCardCoursesProps) => {
           selectedCourses.map((c: ICourseSchedule) => meetingTimeListItems(c))}
         {selectedCourses.length === 0 && (
           <NoItems as="li">
-            <NoItemsImage src={courses} alt="" />
+            <NoItemsImage src={zcourses} alt="" />
```

Comment:
use the `coursesSvg` import recommendation instead, it just makes the import and this line more clear

Reply:
Does this need an `alt` and/or other accessibility needs?

Reply:
made the change to name, not sure if alt is needed. there was no alt when i jumped in, so i just pretended like that was ok. ill try to remember to ask on tuesday when we get back

Reply:
@DeveloperPiatt gotcha.. yea, it'd be best to remove it if it isn't needed or include some text if it should be there